conductor_api/src/interface.rs

```diff
@@ -138,8 +138,18 @@ impl ConductorApiBuilder {
             let hc_lock = instance.clone();
             let hc_lock_inner = hc_lock.clone();
             let mut hc = hc_lock_inner.write().unwrap();
-            let call_params = params_map.get("params");
-            let params_string = serde_json::to_string(&call_params)
+
+            // Getting the arguments of the call contained in the json-rpc 'params'
+            let call_args = match params_map.get("args") {
+                Some(args) => Some(args),
+                None => {
+                    // TODO: Remove this fall back to the previous impl of inner 'params' 
+                    // as soon as its deprecation life cycle is over <17-04-19, yourname> //
+                    hc.context().log("DEPRECATION WARNING: Using 'params' instead of 'args' for a Zome function call is now deprecated.");
```

Comment:
There is a convention of using `<level>/<module>` for log messages, which makes it easier to apply formatting like colors. We only have one `warn` log that I see, but maybe you could add to that convention:

```suggestion
                    hc.context().log("warn/interface: DEPRECATION WARNING: Using 'params' instead of 'args' for a Zome function call is now deprecated.");
```